lib/dbs/mysql.ts

```diff
@@ -2,9 +2,17 @@ import * as mysql from 'mysql';
 import { promisify } from 'util';
 import { SessionProps, StoreData } from '../../types';
 
+const MYSQL_CONFIG = {
+  host: process.env.MYSQL_HOST,
+  database: process.env.MYSQL_DATABASE,
+  user: process.env.MYSQL_USERNAME,
+  password: process.env.MYSQL_PASSWORD,
+  port: process.env.MYSQL_PORT,
```

Comment:
ahhh - I just noticed that we listed the port as optional in the `.env-sample` - https://github.com/bigcommerce/sample-app-nodejs/blob/main/.env-sample#L31

we might have to optionally add this property to the object:
```jsx
{
    ...(port && { port: MYSQL_PORT }),
}
```

Reply:
also, I believe we're using `4 space` indenting in this repo